app/src/test/groovy/gov/hhs/cdc/trustedintermediary/external/slf4j/Slf4jLoggerTest.groovy

```diff
@@ -1,53 +1,36 @@
 package gov.hhs.cdc.trustedintermediary.external.slf4j
 
 import gov.hhs.cdc.trustedintermediary.wrappers.Logger
-import gov.hhs.cdc.trustedintermediary.context.TestApplicationContext
-
-import org.slf4j.event.Level
 import org.slf4j.MarkerFactory
-
+import org.slf4j.event.Level
+import org.slf4j.spi.NOPLoggingEventBuilder
 import spock.lang.Specification
 
-
 class Slf4jLoggerTest extends Specification {
 
     def "getLoggingEventBuilder returns TRACE level builder test"() {
         given:
-        TestApplicationContext.injectRegisteredImplementations()
         def logger = Slf4jLogger.getLogger()
-        def expectedLevel = Level.TRACE
-        def expectedMessage = "Trace message"
 
         when:
-        def logEventBuilder = logger.getLoggingEventBuilder(Logger.Level.TRACE, expectedMessage)
-        def actualLevel = logEventBuilder["loggingEvent"]["level"]
-        def actualMessage = logEventBuilder["loggingEvent"]["message"]
-
+        def logEventBuilder = logger.getLoggingEventBuilder(Logger.Level.TRACE, "Trace message")
         then:
-        actualLevel == expectedLevel
-        actualMessage.toString().contains(expectedMessage.toString())
+        logEventBuilder.getClass() == NOPLoggingEventBuilder
```

Comment:
@basiliskus, I had to radically change the tests for trace and debug.  SLF4J apparently changes what logging event builder it returns depending on what SLF4J provider we use.  It makes sense that a NOP logging event builder is returned for trace and debug since we ask for only the info level and higher.  It is is weird that that an actual logging event builder was returned previously when we used the SLF4J-simple provider and had trace and debug disabled there too.

I also question the need for these tests.  Previously, we put `Slf4jLogger` into the code coverage ignore section.  See `app/build.gradle:134`.

What are your thoughts?

Reply:
I'm ok with removing these tests. When I added them I think I was not clear that it wasn't needed as it is a humble object. I also like the change for SLF4J provider to get a JSON response. It also seems we'd have more control over the logger, which is something that I remember I struggled with when I was trying to configure the simple logger

Reply:
@basiliskus, are you OK removing just the TRACE and DEBUG levels tests, or are you OK with removing this entire test file?  Thoughts?

Reply:
I'm Ok with removing the entire test file unless you think there is any value to keeping it. The only logic that we added is in `getLoggingEventBuilder` which sets the console output color. And adds the FATAL level. Probably that's the only thing worth testing, if anything